contractsSol5/IKyberStorage.sol

```diff
@@ -73,7 +73,8 @@ contract IKyberStorage {
         returns (
             bytes32 reserveId,
             ReserveType resType,
-            bool isFeeAccountedFlags
+            bool isFeeAccounted,
```

Comment:
please have flags in it
for both.

was discussed on our chat with Victor and Mike.

Idea is when we have values for a few reserves. its plural.
and here flags make sense.
or u have better one?

Reply:
As discussed, isFeeAccountedFlag